runtime/gc_base/ObjectAccessBarrier.cpp

```diff
@@ -109,6 +109,97 @@ MM_ObjectAccessBarrier::tearDown(MM_EnvironmentBase *env)
 {
 }
 
+void
+MM_ObjectAccessBarrier::copyArrayCritical(J9VMThread *vmThread, GC_ArrayObjectModel *indexableObjectModel,
+	J9InternalVMFunctions *functions, void **data,
+	J9IndexableObject *arrayObject, jboolean *isCopy)
```

Comment:
I guess these two lines can be coalesced - here and below. Combined line isn't too long